angular-portfolio/src/app/research/research.component.scss

```diff
@@ -0,0 +1,28 @@
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    height: 75vh;
+
+    div {
+        flex-direction: column;
+        width: 33%;
+        padding-left: 5%;
```

Comment:
Recommend using absolute pixel sizes for padding, margins, font-size in this file.